tests/integration/tests.py

```diff
@@ -494,6 +495,13 @@ def test_llama2_7b_chat(self):
             r.launch()
             client.run("vllm_chat llama2-7b-chat".split())
 
+    def test_llava_next_chat(self):
```

Comment:
could we build a pipeline just to test all VLMs?

Reply:
Done